js/test/units/testunitfmt_en_GB.js

```diff
@@ -340,5 +340,209 @@ module.exports.testunitfmt_en_GB = {
         var str = uf.format(m1);
         test.equal(str, "102.338 kilowatt-hours");
         test.done();
-    }
+    },
+    testUnitFormatWithUsagePersonWeight_Short_en_GB: function(test) {
+        test.expect(1);
+        var m1 = MeasurementFactory({
+            unit: "kg",
+            amount: 58
+        });
+
+        var uf = new UnitFmt({
+            usage: "personWeight",
+            measurementSystem: "imperial",
+            length: "short",
+            locale: "en-GB"
+        });
+        var str = uf.format(m1);
+        test.equal(str, "128 lb");
```

Comment:
For England, this should come out as "9.14286 stone". Person weights are in stone. In the US, it is pounds.